src/components/Articles-container.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+import ArticleTile from "./Article-tile"
+import axios from "axios"
+
+function ArticlesContainer () {
+    const [articles,setArticles] = useState([])
+    const [isLoading, setIsloading] = useState(true)
+    
+    useEffect(()=>{
+            axios.get('https://nc-news-v5ne.onrender.com/api/articles').then((result)=>{
```

Comment:
Please extract this functionality to a separate file which will contain all your axios requests. Also please use axios.create() there with base URL. See NC Notes for the (@Axios)